pipelines/jobs/pipeline_job_template.groovy

```diff
@@ -57,7 +57,7 @@ pipelineJob("${BUILD_FOLDER}/${JOB_NAME}") {
         booleanParam('enableTests', runTests, 'If set to true the test pipeline will be executed')
         stringParam('additionalConfigureArgs', "", "Additional arguments that will be ultimately passed to OpenJDK's <code>./configure</code>")
         stringParam('additionalBuildArgs', "", "Additional arguments to be passed to <code>makejdk-any-platform.sh</code>")
-        stringParam('overrideFileNameVersion', "", "REQUIRED for OpenJ9: e.g <code>8u202b08_openj9-0.12.1</code> or <code>11.0.2_9_openj9-0.12.1</code>")
+        stringParam('overrideFileNameVersion', "", "When forming the filename, ignore the build version and override it. For instance if you set this to 'FOO' the final file name will be of the form: <code>OpenJDK8U-jre_ppc64le_linux_openj9_FOO.tar.gz</code>")
```

Comment:
I don't think 'build version' has a context here.  Is 'ignore the part of the filename derived from the publishName or timestamp' more accurate?